packages/validator/src/services/indices.ts

```diff
@@ -101,15 +103,34 @@ export class IndicesService {
     ApiError.assert(res, "Can not fetch state validators from beacon node");
 
     const newIndices = [];
+
+    const allValidators = new MapDef<SimpleValidatorStatus, number>(() => 0);
+
     for (const validatorState of res.response.data) {
+      // Group all validators by status
+      allValidators.set(
+        validatorState.status as SimpleValidatorStatus,
```

Comment:
You will need to map from validator status to simple validator status.
I recommend defining SimpleValidatorStatus separately and independently from validator status.

Something like:
```ts
type SimpleValidatorStatus = "pending" | "active" | "exited" | "withdrawn";
const statusToSimpleStatus: (status: ValidatorStatus): SimpleValidatorStatus => {
  switch (status) {
    case x:
    case y:
    case z:
      return "active";
    ...
  }
}
```

Then here:
```suggestion
        statusToSimpleStatus(validatorState.status),
```

Reply:
Thanks for the suggestion, upon reconsideration is this substantially more readable than [what we had previously?](https://github.com/ChainSafe/lodestar/pull/5161/commits/f1f0e36265ebe457d6230f080db1100f2ecf954b#diff-eb9b1b96c2b62069854c2962a24c1983f831836c63455470fe0d0d78c8f5812cR126-R144) 

Given this will require a mapping and a function to set these values,  I could have just updated the keys in the [previous mapping](https://github.com/ChainSafe/lodestar/pull/5161/commits/f1f0e36265ebe457d6230f080db1100f2ecf954b#diff-eb9b1b96c2b62069854c2962a24c1983f831836c63455470fe0d0d78c8f5812cR126-R130)  to be:

```typescript
const loggedValidatorStates: {[key: string]: ValidatorStatus[]} = {
    "activated": ["active"],
    "pending": ["pending_initialized", "pending_queued"],
    "withdrawn": ["withdrawal_done"],
    "exited": ["exit_slashed", "exit_unslashed"],
  };
```  
and then had:

```typescript
	const statuses = {}

	for (const key in loggedValidatorStates) {
	  const values = loggedValidatorStates[key];
	  let total = 0;
	  for (const value of values) {
	    if (allValidatorStates[value] !== undefined) {
	      total += allValidatorStates[value].length;
	    }
	  }
	  result[key] = total;
	}
    this.logger.info("Validator statuses", {...statuses}); // Output: { activated: 60, pending: 0, .... }
```

I will make these changes now but I think we could have had it either way.